manhatten.js

```diff
@@ -0,0 +1,12 @@
+const botPos = [3, 5];
+const targetPos = [8, -1];
+
+function manhattenDistance(botCoord, targetCoord) {
+  let xAxis = botCoord[0] < targetCoord[0] ?
+              (targetCoord[0] - botCoord[0]) : (botCoord[0] - targetCoord[0]);
```

Comment:
It’s correct but it would be more elegant with `Math.abs()`. Then you don’t need the ternary operator.

Reply:
All done